src/xnnpack/config.h

```diff
@@ -52,6 +52,11 @@ struct xnn_hardware_config {
   // vlenb CSR (VLEN/8). 0 if vector extension is unsupported.
   uint32_t vlenb;
 #endif
+#if XNN_ARCH_PPC64
+  bool use_vsx;
+  bool use_vsx3;
```

Comment:
Where is use_vsx3 used or initialized?

Reply:
Yes, are these fields used anywhere? if not let's remove them until they are needed.